### ISSUE TITLE
phpcs:ignore annotation doesnt work inside a docblock

### ISSUE THREAD

**Issue by edwardspec** [squizlabs/PHP_CodeSniffer#2868]
I'm trying to suppress "Line exceeds 100 characters" warning in the following code:
```php
	/**
	 * @phan-param array{src:string,dst:string,headers?:array<string,string>,ignoreMissingSource?:bool} $params
	 */
	public function something( array $params ) {
	}
```
```
2 | WARNING | Line exceeds 100 characters; contains 108 characters
```

Even with `// phpcs:ignore` before the `@phan-param` line (or at the end of this line) the warning was still emitted.

**Comment by gsherwood** [comment]
The phpcs:ignore annotation (and related ones) were not being detected properly inside docblocks, which is why this wasn't working. The following code will work in 3.5.5:
```php
<?php
/**
 * @phpcs:ignore Generic.Files.LineLength.TooLong
 * @phan-param array{src:string,dst:string,headers?:array<string,string>,ignoreMissingSource?:bool} $params
 */
public function something( array $params ) {
}
```
Thanks for the report.

**Comment by plumthedev** [comment]
hey @gsherwood, that is not working for class method 

I have phpcs 3.5.8
```
❯ vendor/bin/phpcs --version                                                                                                                            
PHP_CodeSniffer version 3.5.8 (stable) by Squiz (http://www.squiz.net)
```

for that code
```
/**
 * @phpcs:ignore Generic.NamingConventions.ConstructorName.OldStyle
 * @return RoleEnum
 */
public function role(): RoleEnum
{
    return $this->role;
}
```

It is still generating error
```
❯ vendor/bin/phpcs --standard=ruleset.xml --tab-width=4 -sp /Users/plumthedev/super-secret-path/Valuing/Role.php
E 1 / 1 (100%)

FILE: /Users/plumthedev/super-secret-path/Valuing/Role.php
-----------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-----------------------------------------------------------------------------------------------------------------------------
 54 | ERROR | PHP4 style constructors are not allowed; use "__construct()" instead
    |       | (Generic.NamingConventions.ConstructorName.OldStyle)
-----------------------------------------------------------------------------------------------------------------------------
```

I'm not sure, but I think is [your code](https://github.com/squizlabs/PHP_CodeSniffer/commit/8909858f9cbfc16ad1b4a2a4b61ff7a3db843251) is covering token only in global indent in Tokenizer and tests

Can you check that? Maybe I make a mistake?

**Comment by daften** [comment]
I can confirm it doesn't work on properties inside classes either. I'll open a new issue.
